cloud/amazon/ec2.py

```diff
@@ -212,7 +212,7 @@
     required: false
     default: 'present'
     aliases: []
-    choices: ['present', 'absent', 'running', 'stopped']
+    choices: ['present', 'absent', 'running', 'rebooted', 'stopped']
```

Comment:
I would argue not to use it in this case because restarting implies a stop / start, which for AWS means the VM is stopped entirely and started on potentially new hardware. A restart can also invoke the termination sequence. Rebooted imho is the best keyword here as it implies a soft (OS) restart.